Marlin/src/pins/rambo/pins_RAMBO_THINKERV2.h

```diff
@@ -34,13 +34,21 @@
   #define FIL_RUNOUT_PIN                      10
 #endif
 
-#if HAS_WIRED_LCD || TOUCH_UI_ULTIPANEL
-  #if ENABLED(ULTIPANEL) || TOUCH_UI_ULTIPANEL
-    #if NONE(VIKI2, miniVIKI)
-      #define BTN_EN1                         64
-      #define BTN_EN2                         63
-    #endif
-  #endif
+// support bltouch and fixed probes
+#if ENABLED(BLTOUCH)
+  #define Z_MIN_PIN    22
+#elif ENABLED(FIX_MOUNTED_PROBE)
+  #define Z_MIN_PIN    4 //24
+#else
+  #define Z_MIN_PIN    10
```

Comment:
This assumes `Z_MIN_PROBE_USES_Z_MIN_ENDSTOP_PIN` which may not always be the case.

Reply:
What's the default config then?